app/src/main/java/no/usn/mob3000/domain/repository/IFriendsRepository.kt

```diff
@@ -0,0 +1,18 @@
+
+
+/**
+ * @author Hussein Abdul-Ameer
+ * created on: 02.11.2024
```

Comment:
As before, use GitHub usernames and ISO8601 conformed dates. Additionally remember to use KDoc string parameter. Remember also that file docstrings should be the first thing in a file, and that the only reason for it to exist is if the file in question needs special considerations. E.g. to inform other developers of something. If not, then use docstrings as usual.